src/site/antora/modules/ROOT/examples/manual/rolling-file/script-condition.groovy

```diff
@@ -0,0 +1,35 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to you under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import java.nio.file.attribute.FileTime
+import java.nio.file.*
+import java.time.ZonedDateTime
+import java.util.stream.Collectors
+
+// tag::script[]
+def limit = FileTime.from(ZonedDateTime.now().minusDays(15).toInstant())
+def matcher = FileSystems.getDefault().getPathMatcher('glob:app.*.log.gz')
+statusLogger.info("Deleting files older than {}.", limit) // <1>
```

Comment:
Is `statusLogger` indeed available? No need to import `StatusLogger` and call `getInstance()`?

Reply:
As I discovered later, all scripts bind `configuration` to a `Configuration` object and `statusLogger` to the `StatusLogger`.

I found that adding status logger statements in scripts makes the life of debugging them much easier.